lib/Client/Util.php

```diff
@@ -49,4 +62,30 @@ public function pushWithMax(&$arr, $item, $max) {
             return false;
         }
     }
+
+    public static function hexdec($input) {
```

Comment:
php style docs for what $input is would be helpful here.

Reply:
done